server/src/cli.rs

```diff
@@ -217,6 +217,10 @@ pub struct EdgeArgs {
     #[clap(long, env, default_value_t = false, requires = "strict")]
     pub streaming: bool,
 
+    /// If set to true. Edge connects to upstream using delta polling instead of normal polling.
```

Comment:
```suggestion
    /// If set to true, Edge connects to upstream using delta polling instead of normal polling.
```

Reply:
Can I suggest putting something in this text to tell users that this is an experimental API and might change?

Reply:
Added